spec/cypress/cypress.env.mco-staging.json

```diff
@@ -0,0 +1,27 @@
+{
+  "baseUrl": "https://mco-staging.dlib.indiana.edu/",
```

Comment:
I think this env file would be better in the mco-staging branch in the IU fork in github.iu.edu.